### PR TITLE
feat: add typings for @ip1sms/is-disposable-phone-number package

### PR DESCRIPTION
**Pull request by ziishaned**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by nilbuild — APPROVED** [review]

**Review by Idnan — APPROVED** [review]

**Review by nilbuild — APPROVED** [review]

File: types/ip1sms__is-disposable-phone-number/ip1sms__is-disposable-phone-number-tests.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import disposable = require('@ip1sms/is-disposable-phone-number');
+
+disposable('');
```
Comment (peterblazejewicz) [inline types/ip1sms__is-disposable-phone-number/ip1sms__is-disposable-phone-number-tests.ts:3]:
```suggestion
disposable(''); // $ExpectType boolean
```

File: types/ip1sms__is-disposable-phone-number/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Type definitions for @ip1sms/is-disposable-phone-number 1.1
+// Project: https://github.com/iP1SMS/is-disposable-phone-number-js#readme
+// Definitions by: Zeeshan Ahmad <https://github.com/ziishaned>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function isDisposablePhoneNumber(phoneNumber: string): boolean;
```
Comment (peterblazejewicz) [inline types/ip1sms__is-disposable-phone-number/index.d.ts:6]:
```suggestion
/**
 * Determine whether a phone number is disposable.
 *
 * @param phoneNumber
 *
declare function isDisposablePhoneNumber(phoneNumber: string): boolean;
```

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@ziishaned thanks!

**Review by Idnan — APPROVED** [review]

**Comment by ziishaned** [comment]
Ready to merge

**Pull request merged**